treeherder/model/models.py

```diff
@@ -1074,7 +1075,8 @@ class FailureLine(models.Model):
     status = models.CharField(max_length=7, choices=STATUS_CHOICES)
     expected = models.CharField(max_length=7, choices=STATUS_CHOICES, blank=True, null=True)
     message = models.TextField(blank=True, null=True)
-    signature = models.TextField(blank=True, null=True)  # Prefix index length 50
+    # Prefix index: signature(50)
+    signature = models.TextField(blank=True, null=True, db_index=True)
```

Comment:
It doesn't do anything (due to the `state_operations` addition above), other than make the defined model conceptually more accurate relative to the actual SQL run.

ie: This commit changes from a "add an index manually that isn't in the model at all" approach to instead: "mark the field as having an index in the model, but override the creation of that index to make it a prefix index instead" approach.

Does that help?